src/onegov/search/__init__.py

```diff
@@ -2,6 +2,8 @@
 log = logging.getLogger('onegov.search')
 log.addHandler(logging.NullHandler())
 
+index_log = logging.getLogger('onegov.search.index')
+
```

Comment:
You should still add the `NullHandler` here too, so you don't get logs for it unless you turn it on via the config file.